.editorconfig

```diff
@@ -4,7 +4,7 @@
 ;
 ; Here are some resources for what's supported for .NET/C#
 ; https://kent-boogaart.com/blog/editorconfig-reference-for-c-developers
-; https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference?view=vs-2017
+; https://learn.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference?view=vs-2017
```

Comment:
nit
```suggestion
; https://learn.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference
```

Reply:
Yep, I checked through all of them last night.

Reply:
great❕